src/apps/src/eryph-packer/Program.cs

```diff
@@ -351,6 +385,39 @@
 
             return packedGenesetInfo;
 
+            async Task AddFodderFromDirectory(DirectoryInfo directory, string architecture)
```

Comment:
Do we support the same fodder file for multiple architectures? I placed `fodder.yaml` in `.\fodder\`, `.\fodder\hyperv\`, `.\fodder\hyperv\amd64\`. I would expect to have 3 packed genes in the end (one for each architecture). Also, when I run the pack command a second time, I get an exception: `System.IO.IOException: Access to the path '...\cmorg\test3\1.0\.packed\778857cedfdfadef12625dc463bf97ec39c8d668132e38c7044c5ed82781dcfb' is denied.` (path truncated).

Reply:
the entire logic was wrong as also the name check loocks for hyperv/amd64 and not for amd64. That and the error above explains the duplicate.